session/session.php

```diff
@@ -442,7 +443,7 @@ function session_status(): int {}
  * @since 5.6
  */
 #[LanguageLevelTypeAware(["7.2" => "bool"], default: "void")]
-function session_abort() {}
+function session_abort():bool {}
```

Comment:
This is not required, since we're using `LanguageLevelTypeAware` here & it's working correctly:
![1](https://user-images.githubusercontent.com/33625946/119516292-f0486900-bd76-11eb-9928-10700579d987.png)
![2](https://user-images.githubusercontent.com/33625946/119516301-f2122c80-bd76-11eb-896c-de6dfe69fcd4.png)

Reply:
Ah, missed it for the abort. Corrected.